src/main/java/com/puppycrawl/tools/checkstyle/SarifLogger.java

```diff
@@ -99,6 +109,12 @@ public class SarifLogger extends AbstractAutomaticBean implements AuditListener
     /** The results. */
     private final List<String> results = new ArrayList<>();
 
+    /** Map to store rule metadata by rule ID. */
+    private final Map<String, ModuleDetails> ruleMetadata = new LinkedHashMap<>();
+
+    /** Map of all available module metadata by fully qualified name. */
+    private Map<String, ModuleDetails> allModuleMetadata;
```

Comment:
```suggestion
    private final Map<String, ModuleDetails> allModuleMetadata = new LinkedHashMap<>();
```
could give SSOT making final.

Reply:
Ok

Reply:
I’m guessing “SSOT” means Single Source of Truth. Would appreciate using full terms in comments — makes it easier to follow.
The change is incorporated.